build.sbt

```diff
@@ -29,10 +33,23 @@ lazy val release = Seq[ReleaseStep](
   pushChanges
 )
 
-lazy val commonSettings = Seq(
+inThisBuild(Seq(
```

Comment:
ThisBuild is the default "Zero" project that it falls back to, so we don't have to add them separately to each project.